importer/helpers/helpers.go

```diff
@@ -110,7 +110,11 @@ func (n *UnixfsNode) AddChild(child *UnixfsNode, db *DagBuilderHelper) error {
 		return err
 	}
 
-	_, err = db.batch.Add(childnode)
+	_, err = db.batch.Add(childnode) //TODO: do we need to provide here?
+	err = db.prov.Provide(childnode.Cid())
```

Comment:
Minor - We should be consistent with how we pass the values returned from `Add` to `Provide`. Here we are ignoring the returned `cid` since it is available on the `childnode`. Elsewhere we use a [separate variable](https://github.com/ipfs/go-ipfs/pull/4333/files#diff-19ceb36d78bde310c54cc44a95bec44cR575) to store the `cid` when calling `Provide`. I am not sure which way is more appropriate, what do you think?

Reply:
We should use `node.Cid()`, Add won't be returning cid after refactor in https://github.com/ipfs/go-ipld-format/pull/8 happens, good catch